osu.Game/Overlays/Settings/SettingsItem.cs

```diff
@@ -80,7 +80,7 @@ public virtual Bindable<T> Bindable
                 controlWithCurrent?.Current.BindTo(bindable);
                 if (ShowsDefaultIndicator)
                 {
-                    restoreDefaultValueButton.Bindable.BindTo(bindable);
+                    restoreDefaultValueButton.Bindable = bindable.GetBoundCopy();

```

Comment:
not a huge fan of non-readonly bindables and these set chains. i think we can refactor this a bit

Reply:
Would probably be a bit easier if we could do it without subclassing `Box`.  That's mostly there so that it can handle the hovering, etc.  Would be nice if we could make `OnHover`, `OnClick`, etc. events.

Reply:
Eh, I don't mind this much since all classes involved are private.